Safewaan/HonkedIn
# C45choi/create forums

## Trello ticket link

(https://trello.com/c/scsvo819/5-as-a-user-i-want-to-be-able-to-create-forums-to-ask-questions-or-discuss-topics)



## Implementation description
* Added the CreateForum.js component
* Created the forums database table



## Steps to test
1. Ensure that the adding of forum title and desc should be submitted
2. Ensure that the entry is added into the DB
3. Ensure that the forum title and desc must be filled 



## What should reviewers focus on?
* Are there any other fields i should add?
